privacyidea/lib/eventhandler/base.py

```diff
@@ -140,6 +141,11 @@ def conditions(cls):
                           "not exist in the userstore anymore."),
                 "value": ("True", "False")
             },
+            CONDITION.TOKEN_IS_EXPIRED: {
+                "type": "str",
+                "desc": _("The token has an expired validity period"),
```

Comment:
I think I am wondering, what timezone you are in! ;-)

It might be interesting one day to distinguish between 
1. token not valid, yet or
2. token not valid anymore.

At the moment tokenclass.check_validity_period() does not distinguish that.
Maybe I should add two additional tokenclass methods like

 * is_token_valid_yet()
 * is_token_still_valid()

Then the event condition could use these methods instead of ``check_validity_period``.

I think we should avoid cryptic numbers in the conditions. We should really use

  * not valid yet  (before the validity period)
  * valid (inside the validity period)
  * not valid (not inside the validity period)
  * not valid anymore (behind the validity period)

I am not a native speaker, so maybe these names are not yet optimal.